lib/ansible/modules/windows/win_psexec.py

```diff
@@ -0,0 +1,159 @@
+options:
+  command:
+    description:
+    - The command line to run through PsExec (limited to 260 characters).
+    required: true
+    default: whoami.exe
+  executable:
+    description:
+    - The location of the PsExec utility (in case it is not located in your PATH).
+    default: psexec.exe
+  hostnames:
+    description:
+    - The hostnames to run the command.
+    - If not provided, the command is run locally.
+  username:
+    description:
+    - The (remote) user to run the command as.
+    - If not provided, the current user is used.
+  password:
+    description:
+    - The password for the (remote) user to run the command as.
+    - This is mandatory in order authenticate yourself.
+  chdir:
+    description:
+    - Run the command from this (remote) directory.
+  noprofile:
+    description:
+    - Run the command without loading the account's profile.
+    default: False
+  elevated:
+    description:
+    - Run the command with elevated privileges.
+    default: False
+  interactive:
+    description:
+    - Run the program so that it interacts with the desktop on the remote system.
+    default: False
+  limited:
+    description:
+    - Run the command as limited user (strips the Administrators group and allows only privileges assigned to the Users group).
+    default: False
+  system:
+    description:
+    - Run the remote command in the System account.
+    default: False
+  priority:
+    description:
+    - Used to run the command at a different priority.
+    choices:
+    - background
+    - low
+    - belownormal
+    - abovenormal
+    - high
+    - realtime
+  timeout:
+    description:
+    - The connection timeout in seconds
+  wait:
+    description:
+    - Wait for the application to terminate.
+    - Only use for non-interactive applications.
+    default: True
+author: Dag Wieers (@dagwieers)
+'''
+
+EXAMPLES = r'''
+# Test the PsExec connection to the local system (target node) with your user (runs whoami)
+- win_psexec:
+
+# Run regedit.exe locally (on target node) as SYSTEM and interactively
+- win_psexec:
+    command: regedit.exe
+    interactive: yes
+    system: yes
+
+# Run the setup.exe installer on multiple servers using the Domain Administrator
+- win_psexec:
+    command: E:\setup.exe /i /IACCEPTEULA
+    hostnames:
+    - remote_server1
+    - remote_server2
+    username: DOMAIN\Administrator
+    password: some_password
+    priority: high
+
+# Run PsExec from custom location C:\Program Files\sysinternals\
+- win_psexec:
+    command: netsh advfirewall set allprofiles state off
+    executable: C:\Program Files\sysinternals\psexec.exe
+    hostnames: [ remote_server ]
+    password: some_password
+    priority: low
+'''
+
+RETURN = r'''
+cmd:
+    description: The complete command line used by the module, including PsExec call and additional options.
+    returned: always
+    type: string
+    sample: 'psexec.exe \\remote_server -u DOMAIN\Administrator -p some_password E:\setup.exe'
```

Comment:
In the old days, the YAML parser tripped over a second colon.
So I still have this weird habit of quoting whatever string with a colon by default.
I only noticed recently that's not needed anymore (with e.g. Windows paths).